### DISCUSSION TITLE
Czech Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#344] [Language Leaders]
@kemo87, @multipm, @zenhas, @halecivo, @jirimissbach - moving discussion about Czech language leader to here.

**Comment by halecivo** [comment]
Hello,

I have several years of experience with Home Assistant, as well as with Google Home. Recently, I successfully configured Rhasspy with my home assistant instance and tested it by creating first Czech command to tell me back time. My work experience is from embedded software development.

I am very happy to see Home Assistant focusing on voice control - I don't like my data being sent to Google and also lack of configuration possibilities for my region in Google Home. That's my motivation to help.

I already merged two PRs  (#237) and (#401). I was also active in other Czech PRs.

**Reply by TomaszPilch** [reply]
@halecivo if you want any help, I'm here to help too. The HA is controlling my home and last year I added google nest and I'm disappointed that it can use only English. I'm looking forward to voice control for my HA.

**Reply by halecivo** [reply]
@TomaszPilch are you on discord?

**Reply by TomaszPilch** [reply]
@halecivo yes Buggies#4588

**Comment by schizza** [comment]
I am able to help too. Over a year experience in home assistant. Did some tutorials connecting thing for HA.
Posted first commit to Czech lang.

**Comment by JakubMacoun** [comment]
If more help is needed, I can help too.
Living with Home Assistant since late 2019. I'm a Java developer, so Python is not really my thing, if that's not a problem.

**Comment by synesthesiam** [comment]
Congrats to @halecivo as our first Czech language leader :slightly_smiling_face:

**Comment by schizza** [comment]
Started [thread](https://github.com/home-assistant/intents/discussions/500) for Czech phrases to support in home assistant voice.

**Comment by synesthesiam** [comment]
@schizza Adding you as a language leader :+1:

**Reply by schizza** [reply]
Thanks!
